server/routes/private.js

```diff
@@ -108,5 +108,25 @@ router.get('/test', async (req, res) => {
     res.send(true);
 });
 
+router.post('/api/add-recurring-schedule', async (req, res) => {
 module.exports = router
 
+    // Validate appropriate parameters are passed to view workers at each school 
```

Comment:
Hi Simran, 
The router should be exported outside of this method as it applies to all private endpoints. I would recommend moving line 112 to line 133.